src/utils/validatePhoneNumbers.test.tsx

```diff
@@ -143,5 +143,10 @@ describe("fullPhoneNumberValidator", () => {
     expect(fullPhoneNumberValidator("+6598261749")).toBe(true);
     expect(fullPhoneNumberValidator("+6598219374")).toBe(true);
     expect(fullPhoneNumberValidator("+6580230000")).toBe(true);
```

Comment:
should we add some of the new numbers that failed the test previously?

Reply:
To add on, i notice the AC mentioned a few cases, don't spot the tests here

```
***Given*** contact number country code is +65
***When*** I enter more than 8 digits in the contact number input field
***Then*** I will see Wrong format error message prompt
```
Add test case for "+65912345789" 

```
***Given*** contact number country code is +65
***When*** I enter a contact number that starts with 0 or 1 or 2 or 4 or 5 or 7 
***Then*** I will see the Wrong format error message prompt```
Add test cases for "+6501234578", "+6511234578", "+6521234578", "+6541234578", "+6551234578", "+6571234578"

```***Given*** contact number country code is +65
***When*** I enter a contact number that contains a special character or space or emoji
***Then*** I will see the Wrong format error message prompt```
Add test cases for "+65😋", "+65@"